src/Controller/Exception.php

```diff
@@ -201,6 +201,10 @@ protected function getContextArray(\Exception $exception = null)
         $loggedOnUser = (bool) $this->app['users']->getCurrentUser() ?: false;
         $showLoggedOff = (bool) $this->app['config']->get('general/debug_show_loggedoff', false);
 
+        // Note: We set this to a high value deliberately. If 'config' is not yet available, the
+        // user can't influence this, so it shouldn't be too low.
+        $traceLimit = (int) $this->app['config']->get('general/debug_trace_argument_limit', 40);
```

Comment:
Same as it does two lines above. I assumed this would work, because i couldn't find a case to trigger it.